velox/docs/functions/spark/decimal.rst

```diff
@@ -0,0 +1,8 @@
+=================
+Decimal functions
+=================
+
+.. spark:function:: unscaled_value(x) -> bigint
```

Comment:
Would you generate the docs locally and verify they look good? I think you need to modify velox/docs/spark_functions.rst to include this new page.

Reply:
Sorry, I haven't added any new rst documents before. I use `./scripts/gen-docs.sh` to generate HTML files for verification, and there are currently no issues.

Reply:
<img width="275" alt="Screenshot 2023-11-07 at 10 09 08 AM" src="https://github.com/facebookincubator/velox/assets/27965151/9748c2db-15d3-4e04-85de-14d5b31a6d2a">

@liujiayi771 Do you see "Decimal functions" appear in this list? Can you share a screenshot?

Reply:
<img width="394" alt="image" src="https://github.com/facebookincubator/velox/assets/13622031/5461e793-5236-4395-9619-8c33e1aa5a0b">
<img width="806" alt="image" src="https://github.com/facebookincubator/velox/assets/13622031/ad524570-b563-4429-adf4-e21691f33b53">